<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IctRcvHookDialog</class>
 <widget class="QDialog" name="IctRcvHookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>回应对话</string>
  </property>
  <widget class="ScrollLineEdit" name="lineEditInfo">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>20</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22px;</string>
   </property>
   <property name="text">
    <string>通话中：1110-2222 陈明适</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>18</y>
     <width>538</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIctRcvHookReturn">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>400</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameVideoShow">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>77</y>
     <width>518</width>
     <height>377</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIctRcvHookVideoOpen">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>272</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>视频开</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIctRcvHookVideoClose">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>272</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>视频关</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIctRcvHookOpen">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>336</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开锁</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonIctRcvHookHangUp">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>208</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>pushButtonIctRcvHookReturn</zorder>
  <zorder>frameVideoShow</zorder>
  <zorder>pushButtonIctRcvHookVideoOpen</zorder>
  <zorder>pushButtonIctRcvHookVideoClose</zorder>
  <zorder>pushButtonIctRcvHookOpen</zorder>
  <zorder>pushButtonIctRcvHookHangUp</zorder>
  <zorder>lineEditInfo</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ScrollLineEdit</class>
   <extends>QLineEdit</extends>
   <header>scrolllineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonIctRcvHookHangUp</tabstop>
  <tabstop>pushButtonIctRcvHookVideoOpen</tabstop>
  <tabstop>pushButtonIctRcvHookVideoClose</tabstop>
  <tabstop>pushButtonIctRcvHookOpen</tabstop>
  <tabstop>pushButtonIctRcvHookReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
